code-studio/src/js/initApp/project.js

```diff
@@ -344,7 +344,7 @@ var projects = module.exports = {
   getStandaloneApp: function () {
     switch (appOptions.app) {
       case 'applab':
-        return 'applab';
+        return appOptions.level.makerlabEnabled ? 'makerlab' : 'applab';
```

Comment:
Yeah, this is reasonable. It would be nice if we had better guarantees about what portions of appOptions were/were not mutable, but I'm pretty sure you dont mutate makerlabEnabled and this should be fine.